index.js

```diff
@@ -1,28 +1,34 @@
+/* eslint-disable no-multiple-empty-lines */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-empty */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-multi-spaces */
+/* eslint-disable quotes */
 /* eslint-disable prefer-destructuring */
```

Comment:
The linters are meant to enforce best practices to ensure that you are writing clean code hence i recommend enabling the linters

Reply:
They were enabled before pushing the code to Git Hub.

Reply:
@DuaneDave Thank you for your comment.  Please reply. Those were momentarily and not entirely. Except you show me where to enable them, I am not sure what you mean.

Reply:
@DuaneDave. Since I have not heard from you. I am resubmitting my pull request to get another feedback

Reply:
I  have now uninstalled the eslint I installed on my VS code from the extensions tap.